content/developers/developer-patterns/scitt-api/index.md

```diff
@@ -0,0 +1,164 @@
+{{< /note >}}
+
+DataTrails supports multiple signing keys.
+_(TODO: link to supported DataTrails Signing Keys\)_<br>
+For a quickstart, we'll create a testing [COSE Key](https://cose-wg.github.io/cose-spec/#key-structure) which DataTrails will cryptographically validate upon registration
+
+1. Create a local signing key
+
+    ```shell
+    openssl ecparam -name prime256v1 -genkey -out scitt-signing-key.pem
+    ```
+
+## Register a SBOM for the Artifact
+
+When registering statements about an artifact, a common identifier is required to correlate a collection of statements.
+In SCITT, a `CWT_Claims subject` property captures the identifier.
+
+The example generates an [SPDX SBOM](https://spdx.dev/), which generates a [Document Namespace](https://spdx.github.io/spdx-spec/v2.2.2/document-creation-information/#65-spdx-document-namespace-field) field which is used for the `CWT_Claims subject` property.
+
+_\<TODO: Add a doc for creating unique identifiers>_
+
+1. Using the [SBOM Tool](https://github.com/microsoft/sbom-tool), generate an SPDX SBOM for `artifact.js`
+
+    ```bash
+    sbom-tool generate -D true \
+      -b scitt/artifact \
+      -ps scitt-community \
+      -pn scitt-nodejs-example -pv 0.0.0 \
+      -nsb https://scitt.io/examples
+    ```
+
+    [More info for the SPDX sbom-tool](https://github.com/microsoft/sbom-tool/blob/main/docs/sbom-tool-arguments.md)
+1. Capture the Document Namespace for the SCITT `subject`
+
+    ```shell
+    SUBJECT=$(cat scitt/artifact/_manifest/spdx_2.2/manifest.spdx.json | jq -r .documentNamespace)
+    ```
+
+1. Sign and Register the SBOM
+
+    ```shell
+    ENTRY_ID=$(statement-register.py \
+      issuer scitt-signing-key.pem \
+      subject $SUBJECT \
+      payload artifacts/_manifest/spdx_2.2/manifest.spdx.json \
+      content-type application/spdx+json)
+    ```
+
+1. Retrieve a SCITT Receipt
+
+    ```shell
+    statement-get-receipt.py $ENTRY_ID
+    ```
+
+## Register an Attestation
+
+Create an Attestation that provides the artifacts compliance to the \<foo> specification
+
+1. Create the Attestation
+
+    ```shell
+    cat > attestation.json <<EOF
+    {
+      "compliance": [
+        {
+          "name": "standard-alpha",
+          "value": "true"
+        },
+        {
+          "name": "standard-beta",
+          "value": "false"
+        }
+      ]
+    }
+    EOF
+    ```
+
+1. Sign and Register the Attestation
+
+    ```shell
+    ENTRY_ID=$(statement-register.py \
+      issuer <identity-reference> \
+      subject $SUBJECT \
+      payload attestation.json \
+      content-type attestation/example )
+    ```
+
+## Retrieve Statements for the Artifact
+
+The power of SCITT is the ability to retrieve the history of statements made for a given artifact.
+By querying the series of statements, consumers can verify who did what and when for a given artifact.
+
+1. Query DataTrails for the collection of statements
+
+    ```shell
+    feed-get.py $SUBJECT
```

Comment:
all our API calls should be done via curl examples.

This one is a list public events call:

```
curl https://app.datatrails.ai/archivist/v2/publicassets/-/events?event_attributes.feed_id={{feed}}
```